### ISSUE TITLE
Investigate 1000-kubemark failure

### ISSUE THREAD

**Issue by wojtek-t** [kubernetes/kubernetes#25134]
There are quite a lot of Density failure in 1000-node kubemark, with

    /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/density.go:563 Expected error: <*errors.errorString | 0xc83cdbc540>: { s: "too high pod startup latency 99th percentil: 5.211839668s", } too high pod startup latency 99th percentil: 5.211839668s not to have occurred

e.g.

    May  2 05:40:46.705: INFO: perc50: 1.502492781s, perc90: 2.282395004s, perc99: 5.159030983s

Examples:
http://kubekins.dls.corp.google.com/view/Scalability/job/kubernetes-kubemark-gce-scale/1114/
http://kubekins.dls.corp.google.com/view/Scalability/job/kubernetes-kubemark-gce-scale/1124/

@gmarek

**Comment by hongchaodeng** [comment]
I have observed the same issue. The throughput (running rate) was about 2 pods/10sec.
The job is still running. I will upload the diagram once it's done.

More info from the trace of kube-scheduler.log:
```
[2.347025ms] [2.347025ms] Computing predicates
[1.598645647s] [1.596298622s] Prioritizing
[5.673612246s] [4.074966599s] Selecting host
```

/cc @xiang90

**Comment by gmarek** [comment]
@hongchaodeng - this is different issue, which will be fixed by #25313.

I looked a bit into the failure @wojtek-t mentioned and it looks to me that for some reason etcd is slow. For this reason I don't think we'll do anything about it and wait for etcd3, as it might be fixed for free.

I'll take a bit closer look to those failures, but that's the current theory.

**Comment by wojtek-t** [comment]
This is about large kubemarks and this is no longer happening. So I'm closing this one.
